api/scpca_portal/test/models/test_library.py

```diff
@@ -0,0 +1,119 @@
+from django.test import TestCase
+
+from scpca_portal.models import Library
+from scpca_portal.test.factories import LibraryFactory, ProjectFactory, SampleFactory
+
+
+class TestGetProjectLibrariesFromDownloadConfig(TestCase):
+    def setUp(self):
+        self.project = ProjectFactory()
+        self.library1 = LibraryFactory(
+            project=self.project, modality=Library.Modalities.SINGLE_CELL, is_multiplexed=False
+        )
+        self.library2 = LibraryFactory(
+            project=self.project, modality=Library.Modalities.SINGLE_CELL, is_multiplexed=True
+        )
+        self.library3 = LibraryFactory(project=self.project, modality=Library.Modalities.SPATIAL)
+
+        self.default_config = {
+            "modality": "SINGLE_CELL",
+            "format": "SINGLE_CELL_EXPERIMENT",
+            "excludes_multiplexed": False,
+            "includes_merged": False,
+            "metadata_only": False,
+        }
```

Comment:
based on our conversation, lets rename this to `download_options` and mutate it directly in each test w/o copying first.